lints/lint_dsa_has_param.go

```diff
@@ -0,0 +1,70 @@
+// lint_dsa_has_param.go
```

Comment:
is there a reason you're putting the name of every file on the top? What does this accomplish?